src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -399,8 +399,8 @@ class FileField extends React.Component {
                     }
                   }}
                   tabIndex="0"
-                  aria-label={`${buttonText} ${uiSchema['ui:title'] ||
-                    schema.title}`}
+                  aria-label={`${buttonText} ${schema.title ||
+                    uiSchema['ui:title']}`}
```

Comment:
Could we use a value like `titleString` here following `buttonText`?

```js

const titleString = typeof uiSchema['ui:title'] === 'string' ? uiSchema['ui:title]' : schema.title
```

Reply:
No worries! I think your suggestion is reasonable. I'll make the change first thing in the morning.